freshmaker/config.py

```diff
@@ -184,19 +184,19 @@ class Config(object):
             'desc': 'Polling interval, in seconds.'},
         'git_base_url': {
             'type': str,
-            'default': "git://pkgs.fedoraproject.org",
+            'default': "git://pkgs.devel.redhat.com",
             'desc': 'Dist-git base URL.'},
         'git_ssh_base_url': {
             'type': str,
-            'default': "ssh://%s@pkgs.fedoraproject.org/",
+            'default': "ssh://%s@pkgs.devel.redhat.com/",
             'desc': 'Dist-git ssh base URL.'},
         'git_user': {
             'type': str,
             'default': '',
             'desc': 'User for git operations.'},
         'git_author': {
             'type': str,
-            'default': 'Freshmaker <freshmaker-owner@fedoraproject.org>',
+            'default': 'Freshmaker <freshmaker-owner@github.com>',
```

Comment:
*Maybe* we should create this user in github too. Or maybe let's just leave it since this conf will never actually be used.